### PR TITLE
Adding experimental rule for detecting first line blank in method block

### PR DESCRIPTION
**Pull request by sowmyav24**
This experimental rule addresses issue [Issue #440](https://github.com/pinterest/ktlint/issues/440).

In addition to checking for first line blank in a method block, it will check for any first blank line in code blocks of the method.Please let me know if the subsequent code blocks in the method need to be ignored from this check.

Due to this new rule, there were violations reported in the code. Have fixed them also.

### CODE REVIEW

**Comment by sowmyav24** [comment]
The pipeline seems to fail with compilation error for the new test classes as it is unable to resolve the dependency. It runs fine in local though.

File: ktlint-ruleset-experimental/src/test/kotlin/com/pinterest/ktlint/ruleset/experimental/NoFirstLineBlankInMethodBlockRuleTest.kt [hunk 1]
```diff
@@ -0,0 +1,111 @@
+package com.pinterest.ktlint.ruleset.experimental
+
+import com.pinterest.ktlint.core.LintError
+import com.pinterest.ktlint.test.format
+import com.pinterest.ktlint.test.lint
+import org.assertj.core.api.Assertions.assertThat
+import org.testng.annotations.Test
```
Comment (jaredsburrows) [inline ktlint-ruleset-experimental/src/test/kotlin/com/pinterest/ktlint/ruleset/experimental/NoFirstLineBlankInMethodBlockRuleTest.kt]: This is the wrong import. You need to use `import org.junit.Test`.

**Comment by jaredsburrows** [comment]
Fix the import and then rebase with master. After that, you should be good.

File: ktlint-ruleset-experimental/src/test/kotlin/com/pinterest/ktlint/ruleset/experimental/NoFirstLineBlankInMethodBlockRuleTest.kt (hunk 1 above)
Comment (sowmyav24) [reply ktlint-ruleset-experimental/src/test/kotlin/com/pinterest/ktlint/ruleset/experimental/NoFirstLineBlankInMethodBlockRuleTest.kt]: All the other tests use this import `import org.testng.annotations.Test` though

Comment (jaredsburrows) [reply ktlint-ruleset-experimental/src/test/kotlin/com/pinterest/ktlint/ruleset/experimental/NoFirstLineBlankInMethodBlockRuleTest.kt]: The do not anymore. Can you please rebase with master?
`git rebase -i origin/master`

Comment (sowmyav24) [reply ktlint-ruleset-experimental/src/test/kotlin/com/pinterest/ktlint/ruleset/experimental/NoFirstLineBlankInMethodBlockRuleTest.kt]: Ah ! Got it.. Will fix it..

**Comment by sowmyav24** [comment]
Fixed all rebase issues and created a PR [#474](https://github.com/pinterest/ktlint/pull/474)

**Pull request closed without merging**